modules/statistics/client/views/statistics.client.view.html

```diff
@@ -6,6 +6,10 @@
       <h2>Trustroots Statistics</h2>
       <br><br><br>
     </div>
+
+    <!-- this doesn't work yet, nginx cors settings - keeping this up here for now so it's quicker to test -->
+    <img src="https://grafana.trustroots.org/render/d-solo/000000002/members?orgId=1&from=1419289200000&to=1543142370362&panelId=1&width=1000&height=500&tz=Europe%2FMadrid">
```

Comment:
Maybe something like:
```html
<div class="panel panel-default">
  <div class="panel-heading">
    <h3 class="panel-title">Member growth</h3>
  </div>
  <div class="panel-body">
    <img class="img-responsive center-block" src="https://grafana.trustroots.org/render/d-solo/000000002/members?orgId=1&from=1419289200000&to=1543142370362&panelId=1&width=1000&height=500&tz=Europe%2FMadrid&theme=light" alt="Member growth">
  </div>
</div>
```

- Added wrapping html panel for prettiness
- Added `&theme=light` to url
- Made image responsive so that it plays nicely with mobile screens

Reply:
Will probably still need to be moved out from the wrapper of stats page since it's quite narrow for an image like this.